libsvmdata/openml.py

```diff
@@ -0,0 +1,69 @@
+import numpy as np
+import openml as oml
+from scipy import sparse
+from .core import _get_data_home, AbstractDataset
```

Comment:
These members don't seem to belong to ``core`` module, they are within ``abstraction``
Also, we better avoid relative imports